resources/design-document.md

```diff
@@ -265,16 +265,30 @@ trainingSeriesName // partition key, string
 
 
 # 8. Pages
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide1.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide2.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide3.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide4.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide5.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide6.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide7.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide8.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide9.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide10.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide11.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide12.PNG)
-![](/home/brecghan/workspace/Unit 7/u7-capstone-Brecghan/resources/Images/Slide13.PNG)
\ No newline at end of file
+### Welcome/Login Screen
+![Slide1.PNG](Images%2FSlide1.PNG)
```

Comment:
Oh I see where I was off on the path before ... these are working correctly, but to make it clearer what's what I'd replace the `%2F` with a `/`. So this would be `Images/Slide1.PNG`. (The `%2F` is the [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding) version of the slash, so it works but in this context isn't needed.)